### ISSUE TITLE
Setup influence function debug process

### ISSUE THREAD

**Issue by GeeYangML** [GeeYangML/Dataset_Debug_Process#1]
1. Prepare dataset
2. Integrate checkpoint saving for each epoch with learning rate
3. Add weight decay to optimizer
4. Use smaller models
5. Train for longer epoch to ensure loss plateau
6. Visualize
